data-avro/src/test/java/com/linkedin/data/avro/TestDataTranslator.java

```diff
@@ -1133,7 +1134,7 @@ private void testDataTranslation(String schemaText, String[][] row) throws IOExc
       {
         // translate from Avro back to Pegasus
         DataMap dataMapResult = DataTranslator.genericRecordToDataMap(avroRecord, recordDataSchema, avroSchema);
-        ValidationResult vr = ValidateDataAgainstSchema.validate(dataMap,
+        ValidationResult vr = ValidateDataAgainstSchema.validate(dataMapResult,
```

Comment:
Why did this need to be changed? Was this a bug in the original testing logic?

Reply:
yes looks like it as we would want to validate the datamap converted from avro not the datamap from test input.